README.md

```diff
@@ -1,4 +1,107 @@
 # Trip-Reviewer
-This project was created to practice my full stack skills. It uses Mongo, React and Node with Express.
+This is a simple blog where each user can post his/her review of some trip so other people could know where it is nice to travel and where it is not so good.
+This project was created to practice my full stack skills. It uses Mongo, React and Node with Express. 
+Also it is my first project where it is implemented my own authentication system with cache and jwt.
+
+The last deployed version is **Trip-Reviewer v1.1.0**. 
+You can check it live at: https://trip-reviewer.herokuapp.com
```

Comment:
These two lines appear at the same line when you look at it in after compilation. It is consistent with the same README of other projects, but I am not sure it looks nice. Maybe would be better to change it everywhere to be on the separate lines.